------------------------------------------------------------------------------------------------------------------------

[2560-12-13 00:48:48 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' has been recovered (tr. 1-6)

[2560-12-13 00:48:48 #2 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 11360@DESKTOP-KCNIK4J

[2560-12-13 00:48:48 #3 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 00:48:56 #4 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 11360@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 01:11:58 #1 store] 
Database 'C:\Users\mix\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 30904@mormix

[2560-12-13 01:11:58 #2 store] 
Configuration path: C:\Users\mix\Documents\GitHub\Lahotel\objectdb\objectdb.conf

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 01:12:02 #1 explorer] 
[ObjectDB 2.7.2_08] javax.persistence.PersistenceException
Attempt to open a database file 'C:\Users\mix\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' that is currently in use by another process (error 141)
	at com.objectdb.Explorer$1.run(Explorer.java:57)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: com.objectdb.o.UserException: Attempt to open a database file 'C:\Users\mix\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' that is currently in use by another process
	at com.objectdb.o.MSG.a(MSG.java:64)
	at com.objectdb.o.LFL.J(LFL.java:930)
	at com.objectdb.o.LFL.x(LFL.java:887)
	at com.objectdb.o.LFL.x(LFL.java:797)
	at com.objectdb.o.DFL.<init>(DFL.java:241)
	at com.objectdb.o.DMG.<init>(DMG.java:27)
	at com.objectdb.o.MST.<init>(MST.java:127)
	at com.objectdb.o.MST.ao(MST.java:109)
	at com.objectdb.o.MSF.Y1(MSF.java:190)
	at com.objectdb.o.OMF.r(OMF.java:783)
	at com.objectdb.jpa.EMF.r(EMF.java:130)
	at com.objectdb.o.OMF.w(OMF.java:694)
	at com.objectdb.jpa.EMF.createEntityManager(EMF.java:149)
	... 17 more

[2560-12-13 01:12:23 #3 store] 
Database 'C:\Users\mix\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 30904@mormix

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 02:28:57 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' has been recovered (tr. 3-32855)

[2560-12-13 02:28:57 #2 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 9412@DESKTOP-KCNIK4J

[2560-12-13 02:28:57 #3 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 02:28:59 #4 type.registry] 
New type com.objectdb.o.LMT

[2560-12-13 02:28:59 #5 type.registry] 
New type com.objectdb.o.VAL

[2560-12-13 02:28:59 #6 type.registry] 
New type com.objectdb.o.RFV

[2560-12-13 02:30:59 #7 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 9412@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 02:57:39 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' has been recovered (tr. 2799-2799)

[2560-12-13 02:57:39 #2 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 3192@DESKTOP-KCNIK4J

[2560-12-13 02:57:39 #3 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 02:57:41 #4 type.registry] 
New type com.objectdb.o.LMT

[2560-12-13 02:57:41 #5 type.registry] 
New type com.objectdb.o.VAL

[2560-12-13 02:57:41 #6 type.registry] 
New type com.objectdb.o.RFV

[2560-12-13 02:58:03 #7 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 3192@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 03:47:09 #1 explorer] 
[ObjectDB 2.7.2_08] javax.persistence.PersistenceException
Attempt to open a database file 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' that is currently in use by another process (error 141)
	at com.objectdb.Explorer$1.run(Explorer.java:57)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: com.objectdb.o.UserException: Attempt to open a database file 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' that is currently in use by another process
	at com.objectdb.o.MSG.a(MSG.java:64)
	at com.objectdb.o.LFL.J(LFL.java:930)
	at com.objectdb.o.LFL.x(LFL.java:887)
	at com.objectdb.o.LFL.x(LFL.java:797)
	at com.objectdb.o.DFL.<init>(DFL.java:241)
	at com.objectdb.o.DMG.<init>(DMG.java:27)
	at com.objectdb.o.MST.<init>(MST.java:127)
	at com.objectdb.o.MST.ao(MST.java:109)
	at com.objectdb.o.MSF.Y1(MSF.java:190)
	at com.objectdb.o.OMF.r(OMF.java:783)
	at com.objectdb.jpa.EMF.r(EMF.java:130)
	at com.objectdb.o.OMF.w(OMF.java:694)
	at com.objectdb.jpa.EMF.createEntityManager(EMF.java:149)
	... 17 more

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 03:47:13 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 5944@DESKTOP-KCNIK4J

[2560-12-13 03:47:13 #2 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 03:47:27 #3 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 5944@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 03:49:48 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 9148@DESKTOP-KCNIK4J

[2560-12-13 03:49:48 #2 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 03:49:56 #3 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 9148@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 03:51:26 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 4492@DESKTOP-KCNIK4J

[2560-12-13 03:51:26 #2 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 03:51:29 #3 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 4492@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 03:54:27 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 9032@DESKTOP-KCNIK4J

[2560-12-13 03:54:27 #2 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 03:54:35 #3 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 9032@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 04:22:29 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 12868@DESKTOP-KCNIK4J

[2560-12-13 04:22:29 #2 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 04:22:41 #3 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 12868@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 06:04:00 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 7092@DESKTOP-KCNIK4J

[2560-12-13 06:04:00 #2 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 06:04:06 #3 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 7092@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 06:31:05 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' has been recovered (tr. 5322-5324)

[2560-12-13 06:31:05 #2 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 9448@DESKTOP-KCNIK4J

[2560-12-13 06:31:05 #3 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 06:31:49 #4 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 9448@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 06:49:51 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' has been recovered (tr. 5325-5325)

[2560-12-13 06:49:51 #2 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 1044@DESKTOP-KCNIK4J

[2560-12-13 06:49:51 #3 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 06:50:09 #4 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 1044@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 07:23:18 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' has been recovered (tr. 1-35648)

[2560-12-13 07:23:18 #2 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 752@DESKTOP-KCNIK4J

[2560-12-13 07:23:18 #3 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 07:23:21 #4 type.registry] 
New type com.objectdb.o.LMT

[2560-12-13 07:23:21 #5 type.registry] 
New type com.objectdb.o.VAL

[2560-12-13 07:23:21 #6 type.registry] 
New type com.objectdb.o.RFV

[2560-12-13 07:23:43 #7 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 752@DESKTOP-KCNIK4J

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 07:28:22 #1 explorer] 
[ObjectDB 2.7.2_08] javax.persistence.PersistenceException
Attempt to open a database file 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' that is currently in use by another process (error 141)
	at com.objectdb.Explorer$1.run(Explorer.java:57)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: com.objectdb.o.UserException: Attempt to open a database file 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' that is currently in use by another process
	at com.objectdb.o.MSG.a(MSG.java:64)
	at com.objectdb.o.LFL.J(LFL.java:930)
	at com.objectdb.o.LFL.x(LFL.java:887)
	at com.objectdb.o.LFL.x(LFL.java:797)
	at com.objectdb.o.DFL.<init>(DFL.java:241)
	at com.objectdb.o.DMG.<init>(DMG.java:27)
	at com.objectdb.o.MST.<init>(MST.java:127)
	at com.objectdb.o.MST.ao(MST.java:109)
	at com.objectdb.o.MSF.Y1(MSF.java:190)
	at com.objectdb.o.OMF.r(OMF.java:783)
	at com.objectdb.jpa.EMF.r(EMF.java:130)
	at com.objectdb.o.OMF.w(OMF.java:694)
	at com.objectdb.jpa.EMF.createEntityManager(EMF.java:149)
	... 17 more

------------------------------------------------------------------------------------------------------------------------

[2560-12-13 07:29:25 #1 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' has been recovered (tr. 35650-35653)

[2560-12-13 07:29:25 #2 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is opened by 10608@DESKTOP-KCNIK4J

[2560-12-13 07:29:25 #3 store] 
Configuration path: C:\Users\terkg\Documents\GitHub\Lahotel\objectdb\objectdb.conf

[2560-12-13 07:29:29 #4 type.registry] 
New type java.lang.Object[]

[2560-12-13 07:29:49 #5 store] 
Database 'C:\Users\terkg\Documents\GitHub\Lahotel\Lahotel\database\lahotel.odb' is closed by 10608@DESKTOP-KCNIK4J

